clang/test/Parser/c2x-typeof-ext-warns.c

```diff
@@ -12,9 +12,10 @@
 // standards before C23, and Clang has followed suit. Neither compiler exposes
 // 'typeof_unqual' as a non-conforming extension.
 
-// Show what happens with the underscored version of the keyword, which is a
-// conforming extension.
+// Show what happens with the underscored version of the keywords, which are
+// conforming extensions.
 __typeof__(int) i = 12;
+__typeof_unqual__(int) u = 12;
```

Comment:
consider adding a test for the non-underscore-suffixed variant as well.

Reply:
Should I add one for `__typeof` as well?

Reply:
No harm in adding that extra coverage as well.

Reply:
Done in 6186b113e9c75c808ff0a08455353d2bb8c3f50b.